code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -463,16 +463,17 @@
 
 	SStun = world.time + rand(20,60)
 
-	mobility_flags &= ~MOBILITY_MOVE
+	Stun(3)
```

Comment:
you sure?

Reply:
Same effect. This removed the ability to move for `0.3` seconds. That's what `Stun()` does.